source/openfisca-web-api/index.md

```diff
@@ -12,23 +12,28 @@ config-openapi
 OpenFisca provides a web API package, compatible with all country packages.
 Using a web interface, app developers can access information and computations without installing anything locally.
 
-## Public France API
+## Demonstration API
 
-The latest version of the France web API is [`api.fr.openfisca.org/latest`](https://api.fr.openfisca.org/latest). Its endpoints are documented in [`fr.openfisca.org/legislation/swagger`](https://fr.openfisca.org/legislation/swagger). This API is provided for prototyping purposes only: it has no SLA (service level agreement, i.e. guarantee of availability) and is kept up to date automatically, with no regard for breaking changes. No support will be provided for this instance. You should not build production systems on top of it.
+The latest version of the OpenFisca Country Template web API can be found at [`api.demo.openfisca.org`](https://api.demo.openfisca.org/). Its endpoints are documented at [`legislation.demo.openfisca.org/swagger`](https://legislation.demo.openfisca.org/swagger). 
+
+## OpenFisca country APIs
+
+Each country/jurisdiction can host an API endpoint and often an OpenFisca instance is deployed multiple times in support of different applications. 
+By way of example, an instance of the French API is available at [api.fr.openfisca.org](https://api.fr.openfisca.org/). It is provided for prototyping purposes only: it has no SLA (service level agreement, i.e. guarantee of availability) and is kept up to date automatically, with no regard for breaking changes. No support is provided for this instance. You should not build production systems against it.
 
 Anyone wanting a stable OpenFisca web API is invited to [host their own](#hosting-an-api-instance).
 
-## Use Cases
+## Example use cases
 
-The following services use the OpenFisca web API:
+The following services utilise the France OpenFisca web API:
 
 - [fr.openfisca.org/legislation](https://fr.openfisca.org/legislation), giving you information on available OpenFisca variables.
 - [Mes Aides](https://mes-aides.gouv.fr), the French social benefits simulator.
 - [PA-comp](https://pa-comp.firebaseapp.com), a divorce fiscal impact simulator.
```

Comment:
This is all outdated. Either drop the section altogether or link to https://openfisca.org/en/showcase/.